antora.yml

```diff
@@ -315,6 +317,8 @@ asciidoc:
         types: ["processor"]
       - name: "redis_streams"
         types: ["input", "output"]
+      - name: "schema_registry"
```

Comment:
@Jeffail. I've put schema_registry into the list of certified components. Is this correct?

Reply:
Yep